packages/client-helper/src/helpers/utils.ts

```diff
@@ -158,7 +158,7 @@ export const parseEvent = (
               const event = clValue.get(CLValueBuilder.string("event_type"));
               if (
                 hash &&
-                hash.value() === contractPackageHash &&
+                hash.value() === contractPackageHash.toLowerCase() &&
```

Comment:
Can you add a comment explaining why we're doing that?

Reply:
done :)